cdk8s/lib/index.js

```diff
@@ -0,0 +1,34 @@
+const cdk8s = require('cdk8s');
```

Comment:
Putting this file here allows `wing compile -t @winglibs/cdk8s` to work (it's hard coded in our platform system).